plugins/create-plugin.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+#
+# Copyright (C) 2023 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+# pylint: disable=invalid-name,missing-docstring,consider-using-f-string
+
+import os
+import argparse
+import glob
+from jinja2 import Environment, FileSystemLoader, select_autoescape
+
+subst = {}
+
+
+def subst_add_string(key: str, value: str) -> None:
+
+    subst[key] = value
+    subst[key.lower()] = value.lower()
+    subst[key.upper()] = value.upper()
+
+
+def subst_replace(data: str) -> str:
+
+    for key, value in subst.items():
+        data = data.replace(key, value)
+    return data
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--vendor", action="store", type=str, default="Hughski", help="Vendor name"
```

Comment:
rather than a default, this should be empty (so it will fail unless it's specified).

Reply:
Agree, done.